containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/enable_xhprof

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+export PATH=$PATH:/usr/sbin:/sbin
+phpdismod blackfire
+mkdir -p ${XHPROF_OUTPUT_DIR}
+phpenmod xhprof
+killall -HUP php-fpm 2>/dev/null || true
```

Comment:
I think it should use `USR2` signal instead of hup

Reply:
It looks like perhaps php-fpm could have ended up with SIGHUP in addition, https://bugs.php.net/bug.php?id=67553, but I don't see traction on that.

Reply:
Thanks for links, this bug is old and as PHP is shipping init.* scripts nobody cares about to add HUP support. Maybe filing PR could help as PHP 8.1 going to feature freeze